package.json

```diff
@@ -58,12 +59,12 @@
     "lint": "npm-run-all --parallel lint:js lint:sass",
     "lint:js": "eslint src",
     "lint:sass": "sass-lint --verbose --no-exit src/**/*.scss",
-    "test": "npm run build && web-ext run -s dist",
     "watch:js": "onchange -p -v 'src/**/*.js' -- npm run bundle:js",
     "watch:css": "onchange -p -v 'src/**/*.scss' -- npm run bundle:css",
     "watch:assets": "onchange -p -v 'src/{manifest.json,icons/*,popup/*.html}' -- npm run copy:assets",
     "storybook": "start-storybook -s ./dist -p 6006",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "test": "npm run lint && echo \"tests TBD\""
```

Comment:
Should we also run the build + package, to make sure that passes?  (And I'm a little surprised building it doesn't package it, or packaging it doesn't build it.)